compiler/src/dotty/tools/dotc/core/SymDenotations.scala

```diff
@@ -99,6 +99,14 @@ trait SymDenotations { this: Context =>
         explain("denotation is not a SymDenotation")
     }
   }
+
+  /** Configurable: Accept stale symbol with warning if in IDE */
+  def acceptStale(denot: SingleDenotation): Boolean =
+    (mode.is(Mode.Interactive) && Config.ignoreStaleInIDE) && {
+      ctx.warning(denot.staleSymbolMsg)
```

Comment:
Such a warning will popup in the IDE (not currently because we ignore warnings without positions, but that's just because of an issue in the LSP (https://github.com/Microsoft/language-server-protocol/issues/249)), maybe use ctx.debug instead ?